molpipeline/mol2any/mol2bool.py

```diff
@@ -0,0 +1,52 @@
+"""Pipeline elements for converting instances to bool."""
+
+from typing import Any
+
+from molpipeline.abstract_pipeline_elements.core import (
+    MolToAnyPipelineElement,
+    InvalidInstance,
+)
+
+
+class MolToBool(MolToAnyPipelineElement):
+    """Element to generate a bool array from input."""
+
+    def __init__(
```

Comment:
Do we need the init if it is not doing anything except call super init? I would have guessed we can go without and save some lines? :-)

Reply:
We have the name which is set via the default value.

Reply:
okay.. why did you not just use self.__class__.__name__ in the init of the super function? And why do you need it here, if we always pass a tuple to the pipeline with (name, class)?

Reply:
That would be possible.

Reply:
your call...

Reply:
I created a new issue to investigate this https://github.com/basf/MolPipeline/issues/60